paasta_tools/automatic_rollbacks/signalfx.py

```diff
@@ -0,0 +1,51 @@
+import time
+from typing import Callable
+from typing import Dict
+from typing import List
+
+from signalfx import SignalFx
+from signalfx.signalflow.messages import DataMessage
+from signalfx.signalflow.messages import MetadataMessage
+
+
+def _convert_sfx_timestamp(ts: int) -> float:
+    '''SignalFx uses millisecond int timestamps, we want floating point seconds'''
+    return float(ts) / 1000
+
+
+def tail_signalfx(query: str, lookback_seconds: float, callback: Callable) -> None:
+    if lookback_seconds > 0:
+        start_timestamp_milliseconds = (time.time() - lookback_seconds) * 1000
+    else:
+        start_timestamp_milliseconds = None
+
+    with open('/nail/home/krall/.signalfx_token') as f:
```

Comment:
er. I'm pretty sure we have this as a system pasta config

Reply:
oh, derp